src/components/MutationErrorBanner/MutationErrorBanner.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropsWithChildren } from "react";
+import { UseMutationResult } from "@tanstack/react-query";
+import ErrorBanner from "../ErrorBanner/ErrorBanner";
+
+interface MutationErrorBannerProps extends PropsWithChildren {
+  // Not clear why the `any` can't be replaced by `unknown` here.
```

Comment:
Thanks for documenting this (in case a future maintainer feels motivated to try to dig into it)!